crates/noirc_evaluator/src/errors.rs

```diff
@@ -94,7 +94,8 @@ impl From<RuntimeError> for FileDiagnostic {
 impl RuntimeError {
     fn into_diagnostic(self) -> Diagnostic {
         match self {
-            RuntimeError::InternalError(_) => {
```

Comment:
maybe you should give more detailed information about the error here?

Reply:
Probably. Looks like there's an empty secondary message we're not using.